config/99-realsense-d4xx-mipi-dfu.rules

```diff
@@ -1,2 +1,6 @@
-SUBSYSTEM=="d4xx-class",KERNEL=="d4xx-dfu[0-9]*", GROUP="video", MODE="0660"
+# Device rules for Intel RealSense MIPI devices.
 
+# DFU rules
+SUBSYSTEM=="d4xx-class",KERNEL=="d4xx-dfu*", GROUP="video", MODE="0660"
+# video links for SDK, binding for ipu6
+SUBSYSTEM=="video4linux", ATTR{name}=="DS5 mux *", RUN+="/bin/bash -c 'rs_ipu6_d457_bind.sh > /dev/kmsg; rs-enum.sh -q > /dev/kmsg'"
```

Comment:
Is it possible to identify we are running on ADL-P and only run this script if needed?
Not sure jetson users want to see IPU6 script call..

Reply:
after looking on debian package for udev rules, we have no way to get that information on debian installation stage.
I decided to merge all paltforms to one script. If it is so crucial - we can add post installation step and remove unnecessary files and modify rules but it will be a mess..